src/java.ts

```diff
@@ -47,6 +47,20 @@ function supportsJreProvisioning(serverUrl: string, serverVersion: SemVer) {
   return supports;
 }
 
+//typeguard temporary for POC
+function isJreMetaData(data: any): data is JreMetaData {
+  return (
+    typeof data === 'object' &&
+    data !== null &&
+    typeof data.type === 'string' &&
+    typeof data.path === 'string' &&
+    typeof data.filename === 'string' &&
+    typeof data.extension === 'string' &&
+    typeof data.md5 === 'string' &&
+    typeof data.javaPath === 'string'
+  );
+}
```

Comment:
NP: I think this code can be dropped

Reply:
indeed, Iwas doing some work with Istvan here that is not needed